### DISCUSSION TITLE
Welcome to gentee Discussions!

### DISCUSSION THREAD

**Discussion by gentee** [gentee/gentee#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mugi4878** [comment]
How can i get code in C#?
I want recorded code in C#

**Reply by gentee** [reply]
Unfortunately, there is not C# version. Gentee is a golang project.

**Reply by Mugi4878** [reply]
Oh! ok, Thanks for your response.
But i have one doubt, it only recording based on points(cursor location) but i need to record the cursor clicked on which window(ex: paint window).

Ex: I recorded on Paint window, when i play the recorded file at the time my system current window is desktop some mismatch found so i want to play that file in recorded window (it only works truly when i set current window is paint window otherwise some mismatch).

**Reply by gentee** [reply]
I think you say about different software (PerfectAutomation). Gentee is a script programming language and it cannot record and play keyboard and mouse actions. PerfectAutomation is not supported now and it is distributed as is.
